lib/sender.js

```diff
@@ -4,7 +4,7 @@
 
 const net = require('net');
 const tls = require('tls');
-const { randomFillSync } = require('crypto');
+const { randomFillSync } = require('./random-util');
```

Comment:
I'm not a huge fan of this kind of api replace. I prefer when reading the code to be noted that this is altered version of the api. I know this will require more changes but states the intent more clearly.

Reply:
randomFillMask